ios/IapSerializationUtils.swift

```diff
@@ -176,6 +176,10 @@ func serialize(_ t: Transaction) -> [String: Any?] {
     ]
 }
 @available(iOS 15.0, tvOS 15.0, *)
+func serialize(_ t: Transaction, _ v: VerificationResult<Transaction>) -> [String: Any?] {
+    return ["verificationResult": v.jwsRepresentation].merging(serialize(t)) { (current, _) in current }
+}
```

Comment:
This could probably be handled in a better way by keeping the verification and transaction separate and skip a merge. However I wanted to avoid introducing new events/bigger changes to the code base. 

If there is a better approach to include the verification result then let me know.

Reply:
Wouldn't it simply be `serialize(t).updateValue(value: v.jwsRepresentation, forKey:"verificationResult")`?

ref: https://stackoverflow.com/a/32249397/570612

Reply:
Looks like it needs to be something like:
```swift
@available(iOS 15.0, tvOS 15.0, *)
func serialize(_ t: Transaction, _ v: VerificationResult<Transaction>) -> [String: Any?] {
    var transaction = serialize(t)
    return transaction.updateValue(v.jwsRepresentation, forKey:"verificationResult") as! [String : Any?]
}
```
Due to to the need of typecasting and resulting immutable value.

Reply:
I think this should work and it's more readable: 
```
@available(iOS 15.0, tvOS 15.0, *)
func serialize(_ t: Transaction, _ v: VerificationResult<Transaction>) -> [String: Any?] {
    var transaction = serialize(t)
    transaction.updateValue(v.jwsRepresentation, forKey:"verificationResult")
    return transaction
}
```

Does that work for you?

Reply:
Yes that works!